src/components/Field.vue

```diff
@@ -169,6 +170,7 @@ export default {
   props: [
     'type',
     'name',
+    'listName',
```

Comment:
I see an extra space at the beginning of this line, not sure if it's my browser?

Reply:
I think that's probably your browser. It's correctly linted and indented from what I see.